widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/GetUnreadMessagesCountWithTimeoutUseCase.kt

```diff
@@ -16,7 +16,7 @@ private const val TAG = "GetUnreadMessagesCountUseCase"
  * @see [GetUnreadMessagesCountWithTimeoutUseCase.invoke]
  */
 @VisibleForTesting
-const val TIMEOUT_SEC = 3L
+internal const val TIMEOUT_SEC = 3L
```

Comment:
this should turn to private in the compiled code

Reply:
But we have disabled the ProGuard optimization. Would the compiler still automatically make it private?

Reply:
Is the `@VisibleForTesting` annotation related to Proguard?

Reply:
This annotation is not related to ProGuard. However as far as I know it does nothing to the access modifier during compilation and is only used for Android Lint inspection:
> You can optionally specify what the visibility should have been if not for testing; this allows tools to catch unintended access from within production code.

Reply:
Indeed, you are correct! I did not know.